### PR TITLE
Improve error messages.

### PR DESCRIPTION
**Pull request by alokmenghrajani**
- [x] @csstaub 
- [x] @mcpherrinm 

Fixes https://github.com/square/keywhiz-fs/issues/16

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -126,7 +127,8 @@ func (c Client) RawSecret(name string) (data []byte, ok bool) {
 		c.Warnf("Secret %v not found", name)
 		return nil, false
 	default:
-		c.Errorf("Bad response code getting secret %v: (status=%v, msg='%v')", name, resp.StatusCode, data)
+		msg := strings.Join(strings.Split(string(data), "\n"), " ")
```
Comment (csstaub) [inline client.go]: Guaranteed that `data` is string? Same below

Comment (alokmenghrajani) [inline client.go]: it's an array of bytes.

Comment (csstaub) [inline client.go]: Sure, but does that byte array actually represent a string?

Comment (mcpherrinm) [inline client.go]: It does if it's one of the "usual" error messages and such from the keywhiz server.  We could escape it somehow so it's more readable but also not putting too much junk in logs.

Comment (alokmenghrajani) [inline client.go]: given that the protocol is http, I think it's fine to expect a string.

**Pull request merged**